app/src/main/java/dev/andrylat/app/App.java

```diff
@@ -0,0 +1,155 @@
+package dev.andrylat.app;
+
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+public class App {
```

Comment:
class also should have correct order to simplify the readability.

order should be something like that:
- public constants
- private constants
- public class fields
- private class fields
- constructors
- public methods
- protected methods
- private methods
- nested classes

Reply:
You should "understand" this order because when you read class from the beginning to the bottom, you should go deeper to the details.